src/app/(auth)/sign-in/page.tsx

```diff
@@ -41,35 +45,59 @@ function SingIn() {
   };
 
   return (
-    <div>
-      <Formik
-        initialValues={iniatialValues}
-        onSubmit={(values) => signInUser(values)}
+    <div className="grid grid-cols-1 md:grid-cols-2 h-screen">
+      <div
+        className="hidden lg:block bg-cover bg-center"
+        style={{ backgroundImage: "url('/images/login-bg.png')" }}
       >
-        {(props) => (
-          <Form>
-            <TextField
-              type="email"
-              name="email"
-              label="Email"
-              placeholder="loremipsum@gmail.com"
-            />
-            <TextField
-              type="password"
-              name="password"
-              label="Password"
-              placeholder="qwert@123"
-            />
-            <button type="submit">Log in</button>
-          </Form>
-        )}
-      </Formik>
-      {isSeller && (
-        <div>
-          logged in
-          <button onClick={() => logOut()}>log out</button>
+        <div className="flex flex-col h-full space-y-0">
```

Comment:
Any reason for having flex-col

Reply:
to set the flex direction to column other-wise it is coming in row